### PR TITLE
Consolidate structured logging release notes

### PR DESCRIPTION
**Pull request by wilsonehusin**
#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:

Consolidate entries related to kubelet structured logging adoption in release notes.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:

/cc ashnehete melodychn pmmalinov01 soniasingla

### CODE REVIEW

File: releases/release-1.21/release-notes/maps/pr-97395-map.yaml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 pr: 97395
 releasenote:
-  text: New Admission Controller "DenyServiceExternalIPs" is made available. Clusters
-    which do not need the service "ExternalIPs" feature should enable this controller.
+  text: New admission controller "DenyServiceExternalIPs" is available.  Clusters
```
Comment (pmmalinov01) [inline releases/release-1.21/release-notes/maps/pr-97395-map.yaml:3]: Should this DenyServiceExternalIPs in a `code block`

Comment (wilsonehusin) [reply releases/release-1.21/release-notes/maps/pr-97395-map.yaml:3]: heh good catch

**Pull request merged**